source/ui/REIXS/REIXSSidebar.cpp

```diff
@@ -86,7 +86,9 @@ REIXSSidebar::REIXSSidebar(QWidget *parent) :
 
 	connect(REIXSBeamline::bl()->valvesAndShutters(), SIGNAL(beamOnChanged(bool)), this, SLOT(onBeamOnChanged(bool)));
 	connect(REIXSBeamline::bl()->mcpDetector(), SIGNAL(countsPerSecondChanged(double)), this, SLOT(onMCPCountsPerSecondChanged(double)));
-
+	
```

Comment:
Hey @dleacock, is this the code from your TFY pull request?  Didn't we already merge this in?